test/Microsoft.TemplateEngine.Mocks/MockFileSystem.cs

```diff
@@ -237,5 +237,15 @@ public DateTime GetLastWriteTimeUtc(string file)
 
             return _files[file].LastWriteTimeUtc;
         }
+
+        public void SetLastWriteTimeUtc(string file, DateTime lastWriteTimeUtc)
+        {
+            if (!FileExists(file))
+            {
+                throw new Exception($"File {file} doesn't exist");
```

Comment:
minor: why not `FileNotFoundException` here? 
https://docs.microsoft.com/en-us/dotnet/api/system.io.filenotfoundexception?view=netcore-3.1

Reply:
It's reasonable to use `FileNotFoundException` here, but this just preserves the behavior of other APIs in this class.  Every API that tests for existence and throws if the file doesn't exist throws an `Exception`.